packages/flame/lib/src/sprite.dart

```diff
@@ -62,6 +62,17 @@ class Sprite {
     src = (position ?? Vector2.zero()).toPositionedRect(srcSize);
   }
 
+  /// Returns a new [Sprite] where the image in memory is just the region
+  /// defined by the original sprite.
+  Future<Sprite> rasterize() async {
+    final rasterizedImage = await toImage();
+    return Sprite(
+      rasterizedImage,
+      srcPosition: Vector2.zero(),
```

Comment:
Isn't this the default?

Reply:
It is the default, but it is not the default directly on the constructor, but set later (hence no lint warning).